controllers/cloud/internal/controller/util/util.go

```diff
@@ -0,0 +1,285 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/go-logr/logr"
+	cloudv1 "github.com/labring/sealos/controllers/cloud/api/v1"
+	cloud "github.com/labring/sealos/controllers/cloud/internal/manager"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	cl "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const ConfigPath = "/etc/config/config.json"
+
+const (
+	TRUE  = "true"
+	FALSE = "false"
+)
+
+func ReadConfigFile(filepath string, logger logr.Logger) (cloud.Config, error) {
```

Comment:
Read config form configmap or ENV is better than config file.

Reply:
Environment variables are set when a container starts and do not update during the container's runtime. Instead, it is better to use ConfigMaps or Secrets as mounted files, which allow for dynamic updates without restarting the container.

Using mounted files from ConfigMaps or Secrets provides a more flexible approach for handling dynamic configuration changes. When these resources are updated, changes can be detected and applied by the application without needing to restart the container. This approach minimizes downtime and ensures a smoother transition when configurations are updated.